src/scripts/bootstrap_trigger/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.9.10
+
+RUN pip install boto3==1.16.33 pandas==1.1.5 synapseclient==2.5.1 pyarrow==4.0.0
+RUN git clone -b https://github.com/Sage-Bionetworks/BridgeDownstream.git /root/BridgeDownstream
```

Comment:
Make sure the docker build doesn't build from an existing cache or new changes to the repo won't get pulled in.